### ISSUE TITLE
PAGE in Transkribus

### ISSUE THREAD

**Issue by kba** [UB-Mannheim/ocr-fileformat#25]
https://github.com/Transkribus/TranskribusPageformat/blob/master/pagecontent_extension.xsd

**Comment by zuphilip** [comment]
Is this extension of the PAGE format still relevant? I see that the latest changes in this [repo](https://github.com/Transkribus/TranskribusPageformat) are from March 2017.

**Comment by bertsky** [comment]
> Is this extension of the PAGE format still relevant? I see that the latest changes in this [repo](https://github.com/Transkribus/TranskribusPageformat) are from March 2017.

It is, but IIUC the schema has moved to https://gitlab.com/readcoop/transkribus/TranskribusCore/-/blob/master/src/main/resources/xsd/pagecontent_extension.xsd

We could now support it as `transkribus` by adding a script based on https://github.com/kba/transkribus-to-prima

**Comment by kba** [comment]
Yes, something like `ocr-transform transkribus-page page2019` for ocr{d_,-}fileformat will be next steps to properly integrate this into OCR-D.

**Comment by stweil** [comment]
Maybe `page-transkribus`? Or `page-readcoop`? Or `page2013t`?

**Comment by bertsky** [comment]
It's 2022 and we are not short of bytes any longer: I recommend against such abbreviations.

Also: `page2013transkribus`

**Comment by stweil** [comment]
We already have cryptical short names like `gcv` (that's why I suggested to extend the help message), but `page2013transkribus` is fine for me, too.
